react/features/virtual-background/components/VirtualBackgroundDialog.js

```diff
@@ -162,7 +164,16 @@ function VirtualBackground({ dispatch, t }: Props) {
                             position = { 'top' }>
                             <Icon
                                 className = { selected === 'blur' ? 'blur-selected' : '' }
-                                onClick = { () => enableBlur() }
+                                onClick = { () => enableBlur(25, 'blur') }
+                                size = { 50 }
+                                src = { IconBlurBackground } />
+                        </Tooltip>
+                        <Tooltip
+                            content = { t('virtualBackground.enableBlur') }
+                            position = { 'top' }>
+                            <Icon
+                                className = { selected === 'less-blur' ? 'blur-selected' : '' }
```

Comment:
Don't we need a different class name for less blur?

Reply:
For now, the behaviour is ok. In this case the single thing that matter is the selected variable.